.github/workflows/main.yml

```diff
@@ -168,4 +172,4 @@ jobs:
         source $CONDA/bin/activate
         conda install anaconda-client
         [[ "$GITHUB_REF" =~ ^refs/tags/ ]] || export LABEL="--label dev"
-        anaconda --verbose --token $ANACONDA_TOKEN upload --user ctools $LABEL conda-bld/*/*.tar.bz2 cb-unix/*/*.tar.bz2 --force
+        anaconda --verbose --token $ANACONDA_TOKEN upload --user ctools $LABEL conda-bld/*/*.tar.bz2 --force

```

Comment:
Not sure why we were still trying to upload `cb-unix/*/*.tar.bz2` here. That's an obsolete need

Reply:
Yeah, true, that's old